addons/dark-www/addon.json

```diff
@@ -2097,9 +2097,9 @@
   "injectAsStyleElt": true,
   "versionAdded": "1.4.0",
   "latestUpdate": {
-    "version": "1.23.0",
-    "isMajor": true,
-    "temporaryNotice": "You can now completely customize the appearance of the Scratch website."
+    "version": "1.33.2",
+    "temporaryNotice": "TODO.",
```

Comment:
I suggest we use three identical temporary notices for each of the 3 addons (easier for translators).

Reply:
Something in the words of: we updated the presets to reflect the color changes by Scratch and added new "blue" presets,